tools/api_key_tool.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import print_function
+import glob
+import re
+import argparse
+
+tokenString = '[Token '
+fixturesDirectory = 'tests/fixtures/'
+zeroapiregex = r'(\[Token )0{40}(\])'
```

Comment:
I like how the regex are all grouped together so that they can be easily compared. Can we name them with consistent `pothole_casing` or `camelCasing` to help with reading? (I.e. `zero_api_regex`, `any_api_regex`